VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseBaseAdapter.java

```diff
@@ -172,6 +144,10 @@ public ViewHolder getTag(View convertView) {
             .findViewById(R.id.bulk_download);
         holder.bulkDownloadVideos = (LinearLayout) convertView
             .findViewById(R.id.bulk_download_layout);
+        holder.rowSubtitlePanel =convertView.findViewById(R.id.row_subtitle_panel);
+        //holder.fullSeparator = convertView.findViewById(R.id.row_full_separator);
+       // holder.halfSeparator = convertView.findViewById(R.id.row_half_separator);
```

Comment:
@aleffert i will use it in next PR. As the UI mockup is inconsistent with separators of list view, commented it out.